### PR TITLE
Finalize edits for JOSS review

### PR DESCRIPTION
**Pull request by lwasser**
this addresses #454 

@betatim what affiliation would you prefer for this submission? earthpy went through pyopensci and now JOSS. i'm adding you as an author to match the zenodo citation!!!
 All i need is your affiliation and we are good!! :)

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -17,93 +17,98 @@ authors:
...
     affiliation: 2
+  - name: Tim Head
+    orcid: 0000-0003-0931-3698
+    affiliation: 3
 affiliations:
   - name: "Earth Lab, University of Colorado - Boulder"
     index: 1
   - name: "University of California - Berkeley, Project Jupyter"
     index: 2
+  - name: "Tim's Affiliation Here"
```
Comment (lwasser) [inline paper.md]: @betatim can you please suggest what you'd like your affiliation(s) to be here?? i can then merge this and earthpy will be in JOSS!!

Comment (lwasser) [inline paper.md]:
```suggestion
  - name: ""Wild Tree Tech""
```

File: paper.md [hunk 2]
```diff
@@ -17,93 +17,98 @@ authors:
...
     affiliation: 2
+  - name: Tim Head
+    orcid: 0000-0003-0931-3698
+    affiliation: 3
 affiliations:
   - name: "Earth Lab, University of Colorado - Boulder"
     index: 1
   - name: "University of California - Berkeley, Project Jupyter"
     index: 2
+  - name: ""Wild Tree Tech""
```
Comment (lwasser) [inline paper.md]:
```suggestion
  - name: "Wild Tree Tech"
```

**Comment by lwasser** [comment]
ok for consistency i'll use the same wild tree tech affiliation that we have for the zenodo citation for the time being!!

**Comment by betatim** [comment]
Wild Tree Tech is good for me

**Comment by lwasser** [comment]
thank you @betatim !!! ok merging this.

**Pull request merged**